packages/models/src/shared/stream.ts

```diff
@@ -1,5 +1,38 @@
 import type { JSONValueType } from './types';
 
+/**
+ * Converts an AsyncIterable<T> to a ReadableStream<T>.
+ *
+ * ReadableStreams implement this natively as `ReadableStream.from` but this is
+ * hardly available in any environments as of the time this was written.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/from_static
+ *
+ * @param iterable Any async iterable object.
+ * @returns A ReadableStream over the iterable contents.
+ */
+export function IterableToStream<T>(iterable: AsyncIterable<T>): ReadableStream<T> {
+  // @ts-ignore
```

Comment:
Why does TypeScript not like this? It seems fine?

Reply:
because typescript does not yet recognize `from` as a static method of readable streams. Which is fair, given this is brand spanking new and only implemented in one or two browsers and nowhere else.